src/browzine-summon-adapter.js

```diff
@@ -1,30 +1,52 @@
-// Begin BrowZine-Summon Integration Code
-angular.module("summonApp.directives")
-.constant("api", "https://api.thirdiron.com/public/v1/libraries/XXX")
-.constant("apiKey", "ENTER API KEY")
-.constant("bookIcon", "https://s3.amazonaws.com/thirdiron-assets/images/integrations/browzine_open_book_icon.png")
-.directive("documentSummary", ["$http", "$sce", "api", "apiKey", "bookIcon", function(http, sce, api, apiKey, bookIcon) {
+browzine.search = (function() {
+  var api = browzine.api;
+  var apiKey = browzine.apiKey;
+
   function isArticle(data) {
-    if(typeof data.document !== "undefined" && data.document !== null) {
-      return data.document.content_type.trim() === "Journal Article";
-    } else {
-      return data.type === "articles";
+    var result = false;
+
+    if(data.document) {
+      if(data.document.content_type) {
+        var contentType = data.document.content_type.trim();
+        result = (contentType === "Journal Article");
+      }
+    }
+
+    if(data.type) {
+      result = (data.type === "articles");
     }
+
+    return result;
   };
 
   function isJournal(data) {
-    if(typeof data.document !== "undefined" && data.document !== null) {
-      return data.document.content_type.trim() === "Journal";
-    } else {
-      return data.type === "journals";
+    var result = false;
+
+    if(data.document) {
+      if(data.document.content_type) {
+        var contentType = data.document.content_type.trim();
+        result = (contentType === "Journal" || contentType === "eJournal");
```

Comment:
This kind of matching and hunting for certain element names we might not know feels a lot like some of the logic we need to do in Crawl Steps in order to find certain elements on a page to click 😊 

One thought: we are doing case-sensitive comparison here, but could we do a case insensitive compare?

Perhaps even better, but more speculative: what if we just look for `journal` to appear, case-insensitively, anywhere in `contentType`?  I'm just thinking if other libraries might call the content type "electronicJournal" or "e-Journal" or something like that... but I don't know!

Reply:
Had the same thought about checking for "journal" when we first started, then ran into this beauty while enhancing article results:
![image](https://user-images.githubusercontent.com/376230/31776432-85979dd4-b4b9-11e7-9927-ae40f040a89d.png)

"Journal Article". . . 😳  - Ok, fair play Summon, fair play. 😂 

No bueno there, but really like the case-insensitive, will add a test and update the same. Pray to the Gods these Summon types aren't user configurable. 😂

Reply:
Oh well look at Summon pulling that kind of thing 😊 

> Pray to the Gods these Summon types aren't user configurable. 😂

Yeah, do we know?  Does @johnseguin or you have a link to any documentation that might mention whether they are or not?